common/artifact-repository/overlays/gcs/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: minio-gateway
+  namespace: $(applicationDefaultNamespace)
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: minio-gateway
+  template:
+    metadata:
+      labels:
+        app: minio-gateway
+    spec:
+      containers:
+        - name: minio-gateway
+          image: minio/minio
+          volumeMounts:
+            - name: gcs-credentials
+              mountPath: /etc/gcs
+              readOnly: true
+          args:
+            - gateway
+            - gcs
+          env:
+            - name: MINIO_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: onepanel
+                  key: artifactRepositoryS3AccessKey
```

Comment:
Are you generating the values for these somewhere? I think for the access key we can possibly use the bucket name but the secret key would def need to be generated.

Reply:
@rushtehrani Yes, in CLI.